plugin.js

```diff
@@ -331,14 +332,34 @@ class CspHtmlWebpackPlugin {
     const scriptShas = this.getShas($, 'script-src', 'script:not([src])');
     const styleShas = this.getShas($, 'style-src', 'style:not([href])');
 
+    const includedScripts = $('script[src]')
```

Comment:
Could you add a `//` comment at the top of this block describing what the lines you have added are doing? It's useful for others who are scanning the file and want to add features around this.

Maybe something like:
```
finding all external script and style files that have been generated by this webpack config, and have been included in this HtmlWebpackPlugin instance's output
```